src/format.js

```diff
@@ -85,13 +85,9 @@ export function formatTime(config, state, value, options = {}) {
     let defaults        = format && getNamedFormat(formats, 'time', format);
     let filteredOptions = filterProps(options, DATE_TIME_FORMAT_OPTIONS, defaults);
 
-    // When no formatting options have been specified, default to outputting a
-    // time; e.g.: "9:42 AM".
-    if (Object.keys(filteredOptions).length === 0) {
-        filteredOptions = {
-            hour  : 'numeric',
-            minute: 'numeric',
-        };
+    if (!filteredOptions.hour && !filteredOptions.minute) {
```

Comment:
`second` should also be considered. While not very useful, browsers support the following:
```js
let df = new Intl.DateTimeFormat('en', {second: 'numeric'});
console.log(df.format(Date.now())); // => "30"
```